docker/Dockerfile.mips64el-unknown-linux-muslabi64

```diff
@@ -19,13 +19,13 @@ RUN /musl.sh \
     TARGET=mips64el-linux-musl \
     "COMMON_CONFIG += -with-arch=mips64r2"
 
-ENV CROSS_MUSL_SYSROOT=/usr/local/mips64el-linux-musl
+ENV CROSS_SYSROOT=/usr/local/mips64el-linux-musl
```

Comment:
this would be a breaking change

I think having both is fine

Reply:
It might be, but we never documented and just used it internally, so I'm not sure if it actually affects anyone. `CROSS_MUSL_SYSROOT` was done in #905, which didn't document it, just made our own logic simpler.